src/Jira/Issues/Walker.php

```diff
@@ -289,19 +294,18 @@ public function count()
 	/**
 	 * Sets callable.
 	 *
-	 * @param callable $callable Callable.
+	 * @param mixed $callable Callable.
 	 *
 	 * @return void
 	 * @throws \Exception When not a callable passed.
 	 */
 	public function setDelegate($callable)
 	{
-		if ( is_callable($callable) ) {
-			$this->callback = $callable;
-		}
-		else {
-			throw new \Exception('passed argument is not callable');
+		if ( !is_callable($callable) ) {
+            throw new \Exception('passed argument is not callable');
 		}
+
+        $this->callback = $callable;
```

Comment:
Code indentation issue. Likely due to mixed tabs/spaces usage.

Reply:
An editorconfig file would prevent this, I added the missing file.